openverse_catalog/dags/common/loader/sql.py

```diff
@@ -345,8 +372,16 @@ def _delete_malformed_row_in_file(tsv_file_name, line_number):
                 write_obj.write(line)
 
 
-def expire_old_images(postgres_conn_id, provider, image_table=TABLE_NAMES[IMAGE]):
-    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+def expire_old_images(
+    postgres_conn_id,
+    task,
```

Comment:
I left a comment below about needing to update tasks that use `op_args` rather than `op_kwargs`, or else the argument unpacking will break here. This one will also need to be updated.

Reply:
Actually this can likely be ignored as this DAG is being retired!